### PR TITLE
[SE-2041] Add documentation on how to reprovision certificates

### PR DESCRIPTION
**Pull request by clemente**
This information could have helped renew certificates some times when the auto-renewal didn't work.

Note that the text doesn't include the exact URLs or port numbers or passwords because they are part of our private infrastructure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,18 @@ You can then install the development requirements using
...
     pipenv run pytest
+
+How to reprovision certificates
+-------------------------------
+
+If a certificate wasn't automatically renewed for some reason, you can trigger a renewal in this way:
+
+1. Log in to the `cert-manager` server as root and run `cerbot delete`, then select the number of the certificate you want to delete.
+
+2. From the consul web interface, navigate to `ocim/certs` and delete the entry corresponding to the old certificate.
```
Comment (itsjeyd) [inline README.md]: @clemente Thanks a lot for adding this info! 💯 

A quick drive-by comment: Could you add a link to the "consul web interface" (even if it's just an example)?

**Comment by giovannicimolin** [comment]
@clemente Can you move this documentation to the private repo at https://gitlab.com/opencraft/documentation/private/blob/master/ops/loadbalancing.md? The other troubleshooting steps for the LB's are there too.

File: README.md (hunk 1 above)
Comment (clemente) [reply README.md]: @itsjeyd Done in the new PR.

**Comment by clemente** [comment]
@giovannicimolin Oh, I didn't know about that file, thanks. It would have helped to find it or to have it linked from here initially.
I would keep all repositories interlinked… but this creates extra effort to maintain the links and many PRs. I'll wait until someone else feels it's needed.

The new PR is at https://gitlab.com/opencraft/documentation/private/-/merge_requests/429
Closing this one.
